package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "devDependencies": {
+    "gulp": "^3.8.8"
+  },
+  "dependencies": {
+    "laravel-elixir": "^3.0.0",
+    "bootstrap-sass": "^3.0.0"
```

Comment:
Actually, why did you even pull this in? I'm confused. It's not even used anywhere?